crates/sui-test-validator/src/main.rs

```diff
@@ -46,11 +50,13 @@ async fn main() -> Result<()> {
         env: Env::NewLocal,
         gateway_address: Some(format!("127.0.0.1:{}", args.gateway_rpc_port)),
         fullnode_address: Some(format!("127.0.0.1:{}", args.fullnode_rpc_port)),
+        websocket_address: Some(format!("127.0.0.1:{}", args.websocket_rpc_port)),
         faucet_address: None,
     })
     .await?;
 
     println!("Fullnode RPC URL: {}", cluster.fullnode_url());
+    println!("Fullnode Websocket RPC URL: http://127.0.0.1:{}", args.websocket_rpc_port);
```

Comment:
```suggestion
    println!("Fullnode Websocket URL: {}", cluster.websocket_address());
```

Reply:
The cluster doesn't have a `websocket_address` exposed, but I can go and add that.